src/mainwindow/mainwindow.cpp

```diff
@@ -2960,16 +2960,14 @@ QString MainWindow::getStyleSheetSuffix() {
     return "fritzing";
 }
 
-void MainWindow::addToMyParts(ModelPart * modelPart, QStringList & m_peAlienFiles)
+void MainWindow::addToMyParts(ModelPart * modelPart, QStringList & peAlienFiles)
 {
```

Comment:
Here, peAlienFiles is a paramater to the method addToMyParts... its scope is limited to the method. So it correctly does not have the m_ prefix. However, since peAlienFiles is not modified, it should be const.